model/distro/distro.go

```diff
@@ -896,6 +896,9 @@ func GetHostCreateDistro(ctx context.Context, createHost apimodels.CreateHost) (
 
 // GetDistrosForImage returns the distros for a given image.
 func GetDistrosForImage(ctx context.Context, imageID string) ([]Distro, error) {
+	if imageID == "" {
+		return nil, errors.Errorf("imageID not found")
+	}
```

Comment:
I feel like you can leave this out, since we don't usually do checks like this elsewhere
```suggestion
```